python/private/pypi/render_pkg_aliases.bzl

```diff
@@ -62,33 +62,65 @@ which has a "null" version value and will not match version constraints.
 """
 
 def _repr_actual(aliases):
-    if len(aliases) == 1 and not aliases[0].version and not aliases[0].config_setting:
-        return repr(aliases[0].repo)
+    if type(aliases) == type(""):
+        return repr(aliases)
 
     actual = {}
-    for alias in aliases:
-        actual[alias.config_setting or ("//_config:is_python_" + alias.version)] = alias.repo
-    return render.indent(render.dict(actual)).lstrip()
+    for alias, repo in aliases.items():
+        if alias.filename:
+            config_setting = """whl_config_setting(\n{}\n)""".format(
+                render.indent("\n".join([
+                    "{} = {},".format(k, repr(v))
+                    for k, v in {
+                        "filename": alias.filename,
+                        "target_platforms": alias.target_platforms,
+                        "version": alias.version,
+                    }.items()
+                    if v
```

Comment:
How about creating text_util.render_kwargs() ? I've had to write similar code to this before. Looks like this same logic is duplicated down below, too.